sdk/python/kfp/compiler/compiler.py

```diff
@@ -778,8 +781,10 @@ def create_workflow(self,
     with dsl.Pipeline(pipeline_name) as dsl_pipeline:
       pipeline_func(*args_list)
 
+    pipeline_conf = pipeline_conf or dsl_pipeline.conf # Configuration passed to the compiler is overriding. Unfortunately, it's not trivial to detect whether the dsl_pipeline.conf was ever modified.
```

Comment:
According to this line, shall we add some doc saying that the one set in pipeline_conf shadows that defined by dsl.get_pipline_conf ?

Reply:
I've added docstring which documents the overriding behavior.